<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Answer</class>
 <widget class="QWidget" name="Answer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>да или нет</string>
  </property>
  <widget class="QPushButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>//назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="Image_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border: 1px solid #000000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../src/like.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Output_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateAnswer_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сформировать ответ!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
